content-src/asrouter/asrouter-content.jsx

```diff
@@ -144,10 +144,6 @@ export class ASRouterUISurface extends React.PureComponent {
     return () => ASRouterUtils.dismissBundle(bundle);
   }
 
-  triggerOnboarding() {
```

Comment:
I'm not seeing the regular onboarding experience show up with both AMO/FXA. I don't think that's intended?